### PR TITLE
metrics: very simple UI

### PR DESCRIPTION
**Pull request by uwedeportivo**
Adds two links to the Admin/Instrumentation page: one that opens Prometheus in a new tab and one that opens Grafana in a new tab.

### CODE REVIEW

**Comment by uwedeportivo** [comment]
The two new env variables SRC_PROMETHEUS_SERVER_URL and SRC_GRAFANA_SERVER_URL will be set in all deploy contexts.

**Comment by uwedeportivo** [comment]
i think i can get the reverse proxy to work. i was pretty close. i'll give it another shot today

**Comment by uwedeportivo** [comment]
i still have to test in the docker cluster and k8s cluster environment so  changes might still come in from there. if you want hold off looking until i got all deploy contexts covered

**Comment by uwedeportivo** [comment]
i added a “Monitor” entry on the site-admin sidebar.
takes you to the grafana dashboard.
i know this is late in the game and not what we agreed on, but it looks nicer than a link in the instrumentation page

image

**Comment by slimsag** [comment]
> i added a “Monitor” entry on the site-admin sidebar. [...]i know this is late in the game and not what we agreed on, but it looks nicer than a link in the  instrumentation page

This looks slick, great work!

File: web/src/site-admin/SiteAdminSidebar.tsx [hunk 1]
```diff
@@ -51,5 +51,8 @@ export const SiteAdminSidebar: React.FunctionComponent<SiteAdminSidebarProps> =
         <a href="/-/debug/" className={SIDEBAR_BUTTON_CLASS}>
             Instrumentation
         </a>
+        <a href="/-/debug/grafana" className={SIDEBAR_BUTTON_CLASS}>
+            Monitor
```
Comment (slimsag) [inline web/src/site-admin/SiteAdminSidebar.tsx]:
```suggestion
            Monitoring
```

**Comment by uwedeportivo** [comment]
with apologies to @lguychard and @felixfbecker i'm going to merge this so it has a wider audience for testing before the 3.8 cut. if anybody is unhappy i can revert back afterwards

**Comment by slimsag** [comment]
Note: I don't think you _need_ their sign-off on this change, the UI portion is extremely minimal and I approved it. If it was a larger change, or there was uncertainty about how it was implemented, then it'd warrant waiting for their review.

**Comment by lguychard** [comment]
Indeed, no need to block on us for such a tiny change!

**Pull request merged**